tests/contrib/boto/test.py

```diff
@@ -80,16 +83,27 @@ def test_s3_client(self):
         s3.create_bucket("cheese")
         spans = writer.pop()
         assert spans
+        eq_(len(spans), 1)
         span = spans[0]
         eq_(span.get_tag(http.STATUS_CODE), "200")
         eq_(span.get_tag(http.METHOD), "PUT")
         eq_(span.get_tag('path'), '/')
         eq_(span.get_tag('aws.operation'), "create_bucket")
 
+        unpatch()
+        s3.get_all_buckets()
+        spans = writer.pop()
+        assert not spans, spans
+
+        # extra patching to check if integration is not wrapped multiple times
```

Comment:
Testing this in that way is fine but usually I prefer having a test duplicated, so that this test (knowing if we trace boto properly) isn't related to a double patch (testing if patching the instance twice will produce 2 traces instead of one).

Is it difficult to split that in two tests? The first one that remains as is, and the second one that simply test that we have one trace if patched 2+ times?